part-2/modules-philsophy-building-blocks.md

```diff
@@ -1,70 +1,70 @@
-# Modules, Philosophy and Building Blocks
+# Software Ecosystem, Modular Philosophy, and Building Blocks Concept
```

Comment:
I tried to rephrase to avoid the term "Modular" or "Modules" in its old definition, because now we mean something else by "modules". So instead of "modular way of software", I use "software ecosystem".